### DISCUSSION TITLE
Unexpected Negative Temperatures in Cement Heat Transfer Simulation

### DISCUSSION THREAD

**Discussion by DanielChou0916** [idaholab/moose#27509] [Q&A Getting Started] [answered]
Hello MOOSE Community,

I'm conducting a heat transfer simulation in cement and encountering an issue where the middle nodes of my domain exhibit negative temperatures. Below are the details of my setup:

Thermal Properties:
Thermal Conductivity: 2.6 W/m°C
Specific Heat: 790 J/kg°C
Density: 2500 kg/m³
Simulation Domain:
The domain consists of eight 2D triangular elements in a square configuration.
Boundary and Initial Conditions:
Initial Temperature: 0°C throughout the domain.
Bottom Boundary: Constant at 0°C.
Top Boundary: Linear increase from 0°C to 25°C over 10 seconds (0.25*t).
Despite these settings, I observed that the temperature at node 3 (located at x,y = (1,1)) drops to -8.92°C, which is unexpected. This issue does not occur when I run a similar simulation in ABAQUS.

Additionally, I attempted to use parameters from an official example, modifying them for my cement material:

Thermal Conductivity: Changed from 45.0 to 2.6 W/m°C
Specific Heat: Unsure if the original 0.5 is in kJ/kg°C.
I would appreciate any insights or suggestions on why these negative temperatures might be occurring and how to resolve this issue. Attached are the mesh file (I_mesh_example_square.e) and the input file (II_2_heat_conduction.i) for reference.

Thank you for your assistance!

My simple simulation:
[heattests.zip](https://github.com/idaholab/moose/files/15163130/heattests.zip)

The MOOSE official example:
[https://mooseframework.inl.gov/modules/combined/tutorials/introduction/thermomech_step01.html](url)

**Comment by GiudGiud** [comment, marked as answer]
Hello

You are very far from a steady state (the linear slope between the two boundaries one will expect) at this point. 
The CFEM discretization in use can present oscillation in the presence of large perturbations, which imposing a dirichlet next to a volume far from that temperature is. 
Do you know what discretization Abaqus uses?

Guillaume

**Reply by DanielChou0916** [reply]
Thank you very much for pointing me out!
I try to add temp from 0~25, and retain 25C to continue the simulation for an extra period
Furthermore, I use finer mesh than my example.
The results make sense to me now!!

And I do not really know what discretization Abaqus uses...
Will pay more attention on these aspects when doing the formal simulation by MOOSE.
But thanks again :)
